shenyu-examples/shenyu-examples-springcloud/pom.xml

```diff
@@ -53,11 +53,11 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
 
- <!--       <dependency>
+        <dependency>
```

Comment:
used spring profiles active nacos , then dependency jar

Reply:
Yeah, now bootstrap avtive is nacos profiles,so use this.